Neos.Neos/Documentation/GettingStarted/FeatureList.rst

```diff
@@ -9,7 +9,7 @@ Feature List
 
 	The following list contains the key technical features of Neos. If you want to learn about the great
 	features and experience that neos offers to editors and visitors please have a look at the `Neos Website
-	<http://www.neos.io>`_.
+	<http://www.neos.io>`_. The `Neos editor documentation<https://www.neos.io/docs-and-support/editor-documentation.html>` is on our website.
```

Comment:
The indention seems weird, does this work as expected?

Reply:
You mean I should have started a new line?

Reply:
I just wondered if it's some kind of special restructuredText-thingy that the link is put on a separate line and - if so - whether it would break things if we put more text on the same line

Reply:
But there was text already before on that line – the dot.